backend/api.test/EndpointTest.cs

```diff
@@ -383,7 +385,23 @@ public async Task AreaTest()
                     W = 1
                 }
             };
+            var testLocalizationPose = new Pose
```

Comment:
Why not just use testPose instead of a separate testLocalizationPose? Alternatively, chance area to also use testLocalizationPose and remove testPose